File: src/CSharp/191.IntHammingWeight.cs
using System;

namespace PeefyLeetCode.IntHammingWeight {
    public class Solution {
        public int HammingWeight (uint n) {
            int count = 0;
            while (n > 0) {
                if ((n & 1) == 1)
                    ++count;
                n >>= 1;
            }
            return count;
        }
    }
}

File: src/Python/101-200/191.IntHammingWeight.py

from copy import deepcopy

class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        return bin(n).count('1')

if __name__ == '__main__':
    solution = Solution()
    print(solution.hammingWeight(11))
    print(solution.hammingWeight(127))
else:
    pass
